src/providers/CourtClock.js

```diff
@@ -17,16 +17,16 @@ function CourtClockProvider({ children }) {
   const { terms = [], termDuration = 0, currentTerm: actualCurrentTerm } =
     courtConfig || {}
 
+  const expectedCurrentTerm = getExpectedCurrentTermId(now, terms, termDuration)
   const { termStartTime, termEndTime } = getTermStartAndEndTime(
-    actualCurrentTerm,
+    expectedCurrentTerm,
     terms,
     termDuration
   )
-  const expectedCurrentTerm = getExpectedCurrentTermId(now, terms, termDuration)
 
   const courtClock = useMemo(
     () => ({
-      currentTermId: actualCurrentTerm,
+      currentTermId: expectedCurrentTerm,
```

Comment:
Not sure what this change does—how does this affect the clock module?

Reply:
This change is to keep track of the supposed current term id assuming the court transitions every time a heartbeat is needed. 

In the context of disputes, we need to be able to transition phases even though the actual current term is behind. 
E.g we are in the committing phase, and the heartbeat service is down (and no one has heartbeat it manually). If we would keep track of the `actualCurrentTerm` as opposed to the `expectedCurrentTerm` then the dispute would not transition to the reveal phase when is supposed to.

> how does this affect the clock module?

The clock module will still work as expected since the UI is reactive relative to the wether the clock is synced or not.